docs/develop/contracts/js/quickstart.md

```diff
@@ -0,0 +1,320 @@
+---
+id: enclave-quickstart
+title: Javascript Enclave
+sidebar_label: Enclave Quickstart
+---
+
+The NEAR platform has historically supported writing contracts in Rust and AssemblyScript. This document aims to introduce developers to a new way of writing smart contracts by using Javascript. 
+
+Javascript is a widely used programming language that is most well known for its Webpage scripting usages. See the [official Javascript docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript) for more details.
+
+<blockquote class="warning">
+<strong>heads up</strong><br /><br />
+
+Javascript smart contract development is not recommended for financial use cases as it is still very new to the NEAR ecosystem.
+
+</blockquote>
+
+## Overview {#overview}
+
+The Javascript Enclave, or jsvm for short, provides an isolated environment where users can learn the basics of how to write smart contracts on NEAR. This isolated environment is run on a virtual machine, similar to how [aurora](https://doc.aurora.dev/getting-started/aurora-engine) operates. Standard smart contracts that are built using Rust or AssemblyScript compile to [WebAssembly](https://webassembly.org/) or simply WASM. With the jsvm, contracts are encoded to base64 and are deployed to a virtual machine.
```

Comment:
The JavaScript Enclave, or JavaScript Virtual Machine (JSVM), . . . 

aurora -> Aurora